### PR TITLE
Remove extra whitespace hiding inside d2l-link elements

### PR DESCRIPTION
**Pull request by j3parker**
This gap was really noticable when a link was beside punctuation (e.g. commas, periods, etc.)

I noticed this when [switching the blog to use Brightspace UI](https://github.com/Brightspace/cdo-blog/pull/179). Here's the before:

d2l-link-wc-inline

and after:

d2l-link-wc-inline

I didn't look closely at how visual-diff works; it looked like something else was maybe supposed to push the goldens to GitHub so I just left it alone. If this fails CI or there is more to do I can check it out on Monday!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ components/*/test/screenshots/golden/
 mixins/*/test/screenshots/current/
 mixins/*/test/screenshots/golden/
 test/sass.output.css
+.DS_Store
```
Comment (j3parker) [inline .gitignore]: Sorry; this is not strictly related. Viewing the golden images in Finder in MacOS dirties the repo with these files.

Comment (dlockhart) [reply .gitignore]: I just have a global ignore rule locally on my machine... otherwise we end up with `.DS_Store`, plus Visual Studio user files, plus Atom user files etc. etc.

**Comment by dlockhart** [comment]
Can you re-run this from a branch instead of a fork? Otherwise the visual diff tests don't run (they need access to encrypted environment variables) and since this will actually produce a new golden we need them to run.

**Comment by j3parker** [comment]
Ah ok. You'll need to give me write permission and/or add me to the org then :)

**Comment by j3parker** [comment]
I'll re-open this as a branch.
